app/javascript/components/home/homeData.js

```diff
@@ -1,25 +0,0 @@
-import React from 'react';
```

Comment:
Moved this to the `Home.jsx` component to match the patterns we had already established there!